src/index.html

```diff
@@ -6,7 +6,40 @@
     <title>Calendar</title>
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
-  <body>
-    <h1>Calendar</h1>
+  <body class="main">
+    <main
+      class="calendar main__calendar--position--center calendar--start-day--sun calendar--month-lenght--29">
+      <div class="calendar__element calendar__element--1"></div>
```

Comment:
```suggestion
      <div class="calendar__element"></div>
```
it would be enough for all days